schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+        "$comment": "TODO: need complete list of language codes",
+        "enum": ["en", "de", "fr", "ru", "cn"]
+      },
+      "active_licenses": {
+        "title": "Active licenses",
+        "description": "Licenses that are selectable upon submission to arXiv.",
+        "type": "string",
+        "enum": [
+          "http://creativecommons.org/licenses/by/4.0/",
+          "http://creativecommons.org/licenses/by-sa/4.0/",
+          "http://creativecommons.org/licenses/by-nc-sa/4.0/",
+          "http://creativecommons.org/licenses/by-nc-nd/4.0/",
+          "http://arxiv.org/licenses/nonexclusive-distrib/1.0/",
+          "http://creativecommons.org/publicdomain/zero/1.0/"
+        ]
+
+      },
+      "inactive_licenses": {
+        "title": "Inactive licenses",
+        "description": "Historical licenses that are no longer selectable upon submission to arXiv.",
+        "type": "string",
+        "enum": [
+          "http://arxiv.org/licenses/assumed-1991-2003/",
+          "http://creativecommons.org/licenses/by/3.0/",
+          "http://creativecommons.org/licenses/by-nc-sa/3.0/",
+          "http://creativecommons.org/licenses/publicdomain"
+        ]
+      },
+      "person": {
+        "type": "object",
+        "properties": {
+          "full_name": {
+            "description": "The fullest representation of the person's name available in arXiv metadata.",
+            "type": "string",
+            "minLength": 1
+          },
+          "last_name": {
+            "description": "The family or surname part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1
+          },
+          "first_name": {
+            "description": "The personal or forename part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1
+          },
+          "suffix": {
+            "description": "The suffix part of the person's name, if available.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "Jr.",
+              "Sr.",
+              "I",
+              "II",
+              "III"
+            ]
+          },
+          "affiliations": {
+            "title": "List of affiliations",
+            "description": "Institutional affiliations",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/affiliation"
+            },
+            "minItems": 1
+          },
+          "orcid": {
+            "title": "ORCID iD",
+            "allOf": [
+              {
+                "type": "string",
+                "pattern": "^\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]$"
+              },
+              {
+                "type": "string",
+                "$comment": "TODO: consider adding 'orcid' format here; would need to implement validator using checksum method described in https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier"
```

Comment:
Not sure you technically an `allOf` to validate both a `pattern` and a `format`. For INSPIRE, we use a custom validator to validate the checksum. We've found it very useful to avoid errors.